tests/test_automation/jenkins_rhea_cpu.sh

```diff
@@ -109,5 +109,14 @@ cp $BUILD_DIR/$BUILD_TAG.o* ../
 
 cp $BUILD_DIR/$BUILD_TAG.o* ../
 
-# explicitly check for correct test output from all builds
+# check for correct test output from all builds
+
+if [ $(grep '100% tests passed, 0 tests failed out of [0-9]*' ../$BUILD_TAG.o* | wc -l) -eq 4 ]
```

Comment:
Sorry, it needs to be -ne 4.

Reply:
sigh, you are right. submitted fix. how embarrassing